requirements.txt

```diff
@@ -8,4 +8,5 @@ dwave-networkx==0.8.13
 dwave-system==1.18.0
 dwave-ocean-sdk==6.3.0
 minorminer==0.2.11
-tqdm==4.64.1
\ No newline at end of file
+tqdm==4.64.1
+pydantic < 2.0
```

Comment:
Instead of constraining Pydantic, I recommend upgrading Ocean, as the issue with Pydantic version bound was fixed in 6.4.1, and then we switched to Pydantic v2 in Ocean 6.6.0. But even 6.6 is over a year old, so upgrading to latest (8.1) would be preferred, unless there's a specific reason to use older Ocean.

Also, it's redundant (and error prone) to pin individual Ocean packages (here: dimod, dwave-system, minorminer), if Ocean is already pinned.